base/applications/osk/main.c

```diff
@@ -1,9 +1,9 @@
-/*
+﻿/*
```

Comment:
???

Reply:
BOM probably

Reply:
Visual Studio 2017 converted the file to Unicode when I added my name to the headers. The change was inevitable.

Reply:
Please convert it to UTF-8 without BOM using an editor that is capable of such a thing.